pkgs/top-level/python-packages.nix

```diff
@@ -16780,6 +16780,25 @@ in modules // {
     };
   };
 
+  simp_le = buildPythonPackage rec {
```

Comment:
`simp_le` is already on master: https://github.com/NixOS/nixpkgs/blob/master/pkgs/tools/admin/simp_le/default.nix